Content.Client/Players/PlayTimeTracking/JobRequirementsManager.cs

```diff
@@ -90,30 +96,33 @@ private void RxJobWhitelist(MsgJobWhitelist message)
         Updated?.Invoke();
     }
 
-    public bool IsAllowed(JobPrototype job, HumanoidCharacterProfile? profile, [NotNullWhen(false)] out FormattedMessage? reason)
+    public bool IsAllowed(RolePrototype role, HumanoidCharacterProfile? profile, [NotNullWhen(false)] out FormattedMessage? reason)
     {
         reason = null;
 
-        if (_roleBans.Contains($"Job:{job.ID}"))
```

Comment:
This is an _incredibly sneaky_ case of `Job:` sneaking out of the Ban database and into real code. Think this is the only instance in master @PJB3005